pyth_observer/events.py

```diff
@@ -185,23 +185,32 @@ class PriceDeviation(PriceValidationEvent):
 
     def is_valid(self) -> bool:
         delta = self.publisher_aggregate.price - self.price.aggregate.price
-        self.deviation = abs(delta / self.price.aggregate.price) * 100
-
-        if (self.price.is_publishing(self.publisher_key) and
-                self.price.is_aggregate_publishing() and
-                self.deviation > self.threshold):
+        if self.price.aggregate.price == 0:
```

Comment:
This is fine solution to the problem here. 

This code does seem suspicious in that an aggregate price of 0 is likely wrong in some way. (I think in devnet you are getting the error because the status is unknown and the price is 0, which is fine.) Maybe leave a TODO here that we will add another alert that validates whether the aggregate price is close to the truth.